features/mock/cantabular_client.go

```diff
@@ -36,6 +38,22 @@ type CantabularClient struct {
 	GetBlockedAreaCountResult        *cantabular.GetBlockedAreaCountResult
 	GetStaticDatasetQuery            *cantabular.StaticDatasetQuery
 	GetStaticDataset                 *gql.Dataset
+	GetObservationsResponse          *cantabular.GetObservationsResponse
+}
+
+// CheckQueryCount implements service.CantabularClient.
+func (c *CantabularClient) CheckQueryCount(context.Context, cantabular.StaticDatasetQueryRequest) (int, error) {
+	if c.BadRequest {
+		return 0, dperrors.New(
+			errors.New("Maximum variables at MSOA and above is 5"),
+			http.StatusBadRequest,
+			log.Data{"errors": map[string]string{"message": "Maximum variables at MSOA and above is 5"}},
+		)
+	}
+	if c.ResponseTooLarge {
+		return 500000, nil
```

Comment:
This is to force a response from the check count that is larger than the maximum number of rows returned.  Generating a json example to mimick this would be too big.